packages/swapper/src/safe-utils/types.ts

```diff
@@ -0,0 +1,56 @@
+import type { ChainId } from '@shapeshiftoss/caip'
+
+export type SafeTxInfo = {
+  transaction: SafeTransactionSuccess | null
+  isSafeTxHash: boolean
+}
+
+export type SafeTransactionSuccess = {
+  safe: string
+  to: string
+  value: string
+  data: string
```

Comment:
optional according to https://safe-transaction-mainnet.safe.global/ and as seen: https://github.com/safe-global/safe-core-sdk/blob/a9e595af13d1b8b8190c7088dcedd8d90b003c27/packages/safe-core-sdk-types/src/types.ts#L214

Reply:
couple others in here if we want to just copy the types over as discussed in dms

Reply:
[badec1269](https://github.com/shapeshift/web/pull/7506/commits/83f4bce6a38ef7b773ae94b115af5ec70bdba25a)